substrafl_examples/strategies_examples/plot_substrafl_fedavg.py

```diff
@@ -391,7 +391,7 @@ def forward(self, x, eval=False):
 # dataset to be stateful.
 
 # Number of model update between each FL strategy aggregation.
-NUM_UPDATES = 15
+NUM_UPDATES = 100
```

Comment:
```suggestion
NUM_UPDATES = 15
```

Is there a reason to change this parameter?

Reply:
Having a performance of 0.6 for Mnsit. This increases the AUC to 0.85 (still bad but better)